cogstat/cogstat_stat.py

```diff
@@ -699,6 +699,61 @@ def variable_pair_standard_effect_size(data, meas_lev, sample=True, normality=No
                                                                                  classes="table_cs_pd"))
     return standardized_effect_size_result
 
+def correlation_matrix(data, x):
```

Comment:
instead of 'x' a more telling variable name could be used, e.g., variables

Reply:
Variable names updated.